govc/dvs/create.go

```diff
@@ -49,9 +51,12 @@ func (cmd *create) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.DVSCreateSpec.ConfigSpec = cmd.configSpec
 	cmd.DVSCreateSpec.ProductInfo = new(types.DistributedVirtualSwitchProductSpec)
 
+	cmd.numUplinkPorts = 2 // Set Default Uplink ports to 2
+
 	f.StringVar(&cmd.ProductInfo.Version, "product-version", "", "DVS product version")
 	f.Var(flags.NewInt32(&cmd.configSpec.MaxMtu), "mtu", "DVS Max MTU")
 	f.StringVar(&cmd.dProtocol, "discovery-protocol", "", "Link Discovery Protocol")
+	f.Var(flags.NewInt32(&cmd.numUplinkPorts), "num-uplinks", "Number of Uplinks")
```

Comment:
Looks like you can just use `f.IntVar(&cmd.numUplinkPorts, ...)` here.  We only need to use `f.Var` + `flags.NewX` for spec fields sent over the wire.  In this case `cmd.numUplinkPorts` is just a local counter.